### PR TITLE
Sharing visualizations

### PR DESCRIPTION
**Pull request by oleurud**
Integration PR

### CODE REVIEW

File: cartoframes/datasets/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from .dataset import Dataset, recursive_read, _decode_geom, get_columns, get_query
+from .dataset_info import DatasetInfo, setting_value_exception
```
[the remark this answers is not in the document]
Comment (oleurud) [reply cartoframes/datasets/__init__.py]: It is used in tests

**Comment by oleurud** [comment]
I have checked examples and everything seems ok (I have only found 2 old references to Dataset).

So, when green, I am going to merge it

**Comment by Jesus89** [comment]
:wave: For next PRs, please wait for our review

**Comment by Jesus89** [comment]
I'll make some comments in this PR https://github.com/CartoDB/cartoframes/pull/741

**Comment by oleurud** [comment]
Ei @Jesus89 sorry if I have seemed rough. 

Everything here has been reviewed before in previous PRs which are incremental (each one using the previous one). In a usual way, I would never request a review of a reviewed code, I have requested a review only to check if I am not breaking anything in the Notebooks, examples and so on and you could be faster than me looking the examples. But I have had time to do it and I have found everything OK.

**Comment by Jesus89** [comment]
Hi @oleurud, don't worry, it's totally fine. Thanks for checking that every notebook works.

Good job, btw!

**Pull request merged**